### DISCUSSION TITLE
Can not edit the "About" section of repository's home page

### DISCUSSION THREAD

**Discussion by Shihab-Shahriar** [community/community#69555] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I'm unable to edit the About section of our public project.

https://github.com/msu-sparta/OpenRAND

**Comment by lucas-0liveira** [comment]
Send the error

**Comment by seyLu** [comment, marked as answer]
#### Hi @Shihab-Shahriar,

You need to have `maintain` or `admin` role in order to be able to edit the about section of your organization repo.

![image](https://github.com/community/community/assets/98249191/4c1a9c23-aaff-48d2-b5f7-024c27a1ddef)

You can read more on:
- [Github docs: Repo roles in org](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization)

**Comment by HarshwardhanPatil07** [comment]
@Shihab-Shahriar 
In organization the owner can assign you role and access of repository that's the reason you are not able to edit the About section of our public project.
